kuba-kostrzewski/simpleToDoApp/src/main/java/pl/kostrzej/simpleToDoApp/components/task/TaskService.java

```diff
@@ -0,0 +1,33 @@
+package pl.kostrzej.simpleToDoApp.components.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.kostrzej.simpleToDoApp.components.user.User;
+import pl.kostrzej.simpleToDoApp.components.user.UserRepository;
+
+import java.util.Date;
+
+@Service
+public class TaskService {
+
+    TaskRepository taskRepository;
+    UserRepository userRepository;
+
+    @Autowired
+    public TaskService(TaskRepository taskRepository, UserRepository userRepository) {
+        this.taskRepository = taskRepository;
+        this.userRepository = userRepository;
+    }
+
+    public User addTask(User user, String title, String description, Date date, boolean done){
+        Task task = new Task();
+        task.setUser(user);
+        task.setTitle(title);
+        task.setDescription(description);
+        task.setDate(date);
+        task.setDate(date);
+        taskRepository.save(task);
+        return userRepository.findByLogin(user.getLogin());
```

Comment:
metoda save zwraca Ci już gotową encję - nie musisz wyszukiwać.

Reply:
Rzeczywiście, tylko metoda addTask zwraca encję User, a save jest robione dla encji Task. Chodziło mi o to żeby po zapisaniu Taska został wykonany SELECT do bazy i zwrócony ten sam User ale ze zaktualizowaną listą Tasków.